style.css

```diff
@@ -30,6 +30,8 @@ body {
 p.text-white {
     color: #fff;
     line-height: 1.8em;
+    padding-top: 2rem;
```

Comment:
Nice to see rems! 👍 .
You could combine these paddings in one declaration, like so: `padding: 2rem 0`. The first part sets top & bottom, the second part: left & right`. :)